config/default.yml

```
@@ -83,6 +83,15 @@ Rails/ActiveSupportAliases:
   Enabled: true
   VersionAdded: '0.48'
 
+Rails/AddColumnComment:
+  Description: >-
+    This cop enforces the use of the `comment` option when creating a new column
```

```suggestion
    This cop enforces the use of the `comment` option when adding a new column
```
(to be consistent with the cop name).